package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "nomios-web",
```

Comment:
Missing `description`, `keywords` (copy it from the README) and also other fields, such as repository and so on. Please check this package.json: https://github.com/ipfs-shipyard/js-human-crypto-keys/blob/master/package.json

Reply:
I've added `homepage` as `https://github.com/ipfs-shipyard/nomios-web#readme`, but I'll update this field as soon as we get the domains.